src/index.html

```diff
@@ -1,34 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Daily Prebuilt with Kahoot</title>
-  <link rel="stylesheet" href="./style.css" />
+  <title>Daily Prebuilt + Kahoot = <3</title>
```

Comment:
Just to make it a little fun and also match the style we used for the Miro + CometChat demo :))